catalog/spatial/registry/registry-identification-plugin/src/test/java/org/codice/ddf/registry/identification/IdentificationPluginTest.java

```diff
@@ -220,31 +225,102 @@ public void testDuplicateCheckingAfterDelete() throws Exception {
         idp.process(new CreateRequestImpl(sampleData));
     }
 
-    @Test
-    public void testUnusedMethods() throws Exception {
-        UpdateRequest updateRequest = new UpdateRequestImpl("abc123", sampleData);
-        assertThat(updateRequest, equalTo(idp.process(updateRequest)));
-        DeleteRequest deleteRequest = new DeleteRequestImpl("abc123");
-        assertThat(deleteRequest, equalTo(idp.process(deleteRequest)));
-        UpdateResponse updateResponse = new UpdateResponseImpl(updateRequest, null, null);
-        assertThat(updateResponse, equalTo(idp.process(updateResponse)));
-    }
-
     private String convert(String path) throws Exception {
         InputStream inputStream = getClass().getResourceAsStream(path);
         BufferedReader buffer = new BufferedReader(new InputStreamReader(inputStream));
         return buffer.lines()
                 .collect(Collectors.joining("\n"));
     }
 
+    @Test(expected = UnsupportedOperationException.class)
```

Comment:
If you change the UnsupportedOperationException to a PluginExecutionException this test will need to change.